vis/Code/SampleWindow.js

```diff
@@ -16,7 +16,7 @@ SampleWindow = (function()
         this.Visible = true;
 
         // Create a grid that's indexed by the unique sample ID
-        this.Grid = this.Window.AddControlNew(new WM.Grid(0, 0, 380, "calc( 100% - 17px )"));
+        this.Grid = this.Window.AddControlNew(new WM.Grid());
```

Comment:
I'm concerned this loses the ability for the grid control to resize with its container. The goal was to allow resizable columns but I never got around to that. Is it much of an issue? I will have to test!

Reply:
I removed these arguments simply because they are no longer internally required in the grid. It looks like resizable columns aren't entirely straightforward (without dragging in libraries like jquery), but i don't think you'd need absolute sizes either way.

Reply:
Definitely not bringing in jquery but you're right about not needing absolute sizes.